sceptre/resolvers/project_variables.py

```diff
@@ -24,6 +26,12 @@ def resolve(self):
         :returns: Value at argument location.
         :rtype: str
         """
+        init(autoreset=True)
```

Comment:
`autoreset` is a really nice feature, which I didn't actually know about. However, we already run `colorama.init()` in [cli.py](https://github.com/cloudreach/sceptre/blob/master/sceptre/cli.py#L123) - could you remove the `init()` from here and add a `Style.RESET_ALL` at the end of the warning?